### DISCUSSION TITLE
How the payment transfer look like for ISV?

### DISCUSSION THREAD

**Discussion by celine3650** [filecoin-project/motion#133] [General]
Payment options:
Out-of-band payment, draft contract like what S3 did
Through Fil+ (if there is a streamlined way to do it)
On-chain payment (Filecoin, other web3 payment, or fiat)

Based on the conversation with DWS, at a minimum they simply want to be able to pay the SPs for deals in FIL automatically using FIL. They'd like the FIL to be deducted from their Motion wallet

**Comment by masih** [comment]
@celine3650 It is unclear if there is anything actionable from engineering perspective here. Removing Beta label in the meantime
